src/loader.js

```diff
@@ -362,7 +362,7 @@ function setupModuleLoader(window) {
           component: function(name, options) {
             function factory($injector) {
               function makeInjectable(fn) {
-                if (angular.isFunction(fn)) {
+                if (isFunction(fn) || isArray(fn)) {
```

Comment:
@petebacondarwin in 99% of the cases (that is when people use angular and not just the loader), `isFunction` comes from `src/Angular.js` and I doubt that there is a need to explain this. For the 1% of cases that the loader is used stand-alone, then it comes from another place.